.github/workflows/gating.yaml

```diff
@@ -25,18 +25,20 @@ jobs:
         # We need to install git inside the container otherwise the checkout action will use Git
         # REST API and the .git directory won't be present which fails due to setuptools-scm
         apt-get update && apt-get install --no-install-recommends --no-install-suggests -y git
-        python3 -m pip install --upgrade pip
-        pip install tox
+        python3 -m venv /var/tmp/venv
+        /var/tmp/venv/bin/pip3 install --upgrade pip
+        /var/tmp/venv/bin/pip3 install nox
+        /var/tmp/venv/bin/pip3 install tomli
```

Comment:
why are you installing it in a venv in a Container, is it because Debian complains about installing to system-wide locations?

Reply:
Have you considered installing to `/usr/local` instead? I'm asking because installing to a venv in a throwaway CI container feels like rubbing the wrong way, but more importantly, I don't know how these containers are set up, IOW `/tmp` is more often than not a `tmpfs` with a fairly low size, you have `/var/tmp` which I think should be fine, but I think with these CI infras using a system-wide location for package installs is a safer bet. Not a blocker though, it's just that this caught my eye.

Reply:
fixed